cli/pom.xml

```diff
@@ -16,7 +16,7 @@
   <properties>
     <!-- TODO: remove once Medium level is set in the core (JENKINS-36716) -->
     <findbugs.threshold>Medium</findbugs.threshold>
-    <junit.jupiter.version>5.5.2</junit.jupiter.version>
+    <junit.jupiter.version>5.7.0</junit.jupiter.version>
```

Comment:
Can we move the version into the parent POM so it is managed in just one place?

Reply:
addressed in eff990fe05